applications/PoromechanicsApplication/tests/element_tests/undrained_soil_column_2D/undrained_soil_column_2D.mdpa

```diff
@@ -5,6 +5,8 @@ Begin Table 1 TIME FACE_LOAD_Y
 End Table
 
 
+Begin Properties 0
+End Properties
 Begin Properties 1
```

Comment:
You can remove these properties if not used

Reply:
I am actually using them for the Conditions (look at the description of the PR). 
The thing is that my “simple” problemtype is prepared to obtain the properties of the faces of the mesh (conditions) from the elements making it easy to write the properties without the need of any specific submodelpart... If I had to write those condition properties associated to a submodelpart, I would need to create that submodelpart first...